### DISCUSSION TITLE
Hide Links of Non-installed/Disabled Components from the SoC's Menu

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#1571] [Ideas] [answered]
If I don't install or later disabled TheHive, for example, the link should be disappeared from the SoC's menu.
(Or options to toggle on/off the links)

**Comment by dougburks** [comment, marked as answer]
I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/1643 for this.
